Content.Shared/Interfaces/GameObjects/Components/Interaction/IHotItem.cs

```diff
@@ -12,6 +12,6 @@ namespace Content.Shared.Interfaces.GameObjects.Components
     /// </summary>
     public interface IHotItem
     {
-        bool IsCurrentlyHot();
+        public bool IsCurrentlyHot();
```

Comment:
is this needed? iirc you can just leave the public modifier out as its implicit. just a nitpick